doc/ug/install_instructions/index.md

```diff
@@ -4,30 +4,83 @@ title: Install Build Requirements
+* A reasonably powerful PC.
   Using a virtual machine can work, but will slow down builds considerably.
   8 GB of RAM or more are highly recommended.
-* Physical access to that machine, root permissions and a graphical environment.
-* Python 3.5.2 or newer. Python 3.6+ is recommended.
-* A C++14 capable compiler. GCC 5 or Clang 3.5 should meet this requirement.
-* 60 GB or more of disk space.
-  EDA tools like Xilinx Vivado can easily take up 40 GB each.
-* We develop and test on the following Linux distributions:
-  * Ubuntu 16.04 LTS
-  * Debian testing
-  * openSUSE Tumbleweed
-  * TODO: Check RHEL/CentOS and SLES (used in many commercial environments)
-
-TODO: Be more specific about the system requirements, especially the Linux distribution.
+* Physical access to that machine, root permissions and a graphical environment are assumed in this guide.
+  However, with an appropriate setup, none of these things are strictly needed.
+* 60 GB or more of disk space, depending on the used EDA tools.
+  (EDA tools like Xilinx Vivado can easily take up 40 GB each.)
+* Linux
+  * Ubuntu 16.04 LTS is the recommended reference platform.
+    Our continuous integration setup runs on Ubuntu 16.04 LTS, which gives us the most confidence that this distribution works out of the box.
+  * We do our best to support other Linux distributions.
+    However, we cannot guarantee they can be used "out of the box" and might require updates of packages.
+    Please file a [GitHub issue](https://github.com/lowRISC/opentitan/issues) if you need help or would like to propose a change to increase compatibility with other distributions.
+    * RedHat Enterprise Linux (RHEL)/CentOS 7 and 8
+    * Many of our contributors use rolling releaese distributions, which generally include the latest stable releases of all software packages, such as Debian unstable, openSUSE Tumbleweed, or Arch Linux.
+
+The following software packages are required to build and use the software, hardware, and tools in the OpenTitan repository.
+Not every interaction with the repository requires all tools -- however, we recommend users installing all dependencies listed below for a seamless development experience.
+
+Note: Version numbers given below indicate a minimum version, newer versions are generally expected to work, unless explicitly stated.
+For packages listed below without a version number we have not determined a minimum version.
+
+* git
+* Python 3.5.2 with pip.
+  Additional Python dependencies are installed through pip.
+* A C++14 capable compiler.
+  GCC 5 or Clang 3.5 should meet this requirement.
+* clang-format
+* [ninja](https://ninja-build.org/) 1.5.1
+* Bash
+* curl
+* xz-utils
+* lsb-release
+* srecord
+* GNU make
+* pkg-config or pkgconf
+* libelf with development headers
+* libftdi with development headers
+* OpenSSL 1.0 or 1.1 with development headers
+* libusb 1.0
+* OpenOCD
+* [Verible](https://github.com/google/verible) {{< tool_version "verible" >}}
```

Comment:
I like the `tool_version` shortcode! Would it make sense to use it for `ninja` too?